src/java.desktop/share/classes/javax/swing/plaf/basic/BasicOptionPaneUI.java

```diff
@@ -478,8 +479,11 @@ public Dimension getPreferredSize() {
                 }
                 // Prevent recursion of more than
                 // 200 successive newlines in a message
+                // and indicate message is truncated via ellipsis
                 if (recursionCount++ > 200) {
                     recursionCount = 0;
+                    addMessageComponents(container, cons, new String("..."),
```

Comment:
Hm. What was the reason to use `new String` constructor here? Is it some security issue?
Seems redundant to me.